src/hpc-cache/README.rst

```diff
@@ -1,5 +0,0 @@
-Microsoft Azure CLI 'hpc-cache' Extension

```

Comment:
Why deleting this file?

Reply:
Because we already have `README.md`, so it is useless.